maplibre-gl-js-vue-amplify-compare-maps/src/aws-exports.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable */
+// WARNING: DO NOT EDIT. This file is automatically generated by AWS Amplify. It will be overwritten.
+
+const awsmobile = {
```

Comment:
This file should never be committed to version control and definitely should not be part of this repo, please remove it asap.

Reply:
Done

Reply:
Done